tutorials/telemetry_hello_world.kp/orig_src/Telemetry Hello World.ipynb

```diff
@@ -279,133 +352,257 @@
2U/lHojDPxDRDzYIn7XCYitgDsj4p6B99cGdo+I77eJP8b8WieU\nSinvBU7t4qK3lLI9ee7aEriEZc9dzwBuBF4SEdc0iP2ecYqOAo4ktysR8ZkGsbcnz7XrA38EtgLO\nAmYDTyP3xTe02RfHmOcwtmVn67vG/wvgc8AVwA7kzfsXgP8AFgNvAQ6NiE83iN3lftjZck9y/rsA\nl0fEwxp89xLgoxFxZinlleS+dyawANgReDmwb0ScOeRl7uRmuJSyLfV3LiL+bwjxdmP58/nFEXHp\ndIxdSvke+Rt8ainl2cD5wNUs3Z5zgb2aJE5LKfsCpwB3ktdA+wDfAH5C7ud7AW+JiK+1WP7Oknhd\nxS6lbAr8F3nuXlLj/RR4LLAJcFREvK/FcneWBCulzCXP6VuTScEXktt0J6CQ9wXPavm7MSsiloz1\nPrBVRIw0iLn3OEWnkdfRvwGIiG9NNfbqYKYlfEopvaT0v5EVO24ecvzTgK/V8+LOwPeAqPN8XP17\njyYVHOr91ZeA/Wqc22vRo8lj6GTyIdjCFsv/sIhY3Pf6GeTxf3HvgUqDmGsDHwP2rct8TET8W1/5\nZsDNTa4r6vdXauJ0YN4bAvPb3BN1tM472w8H5rMWsDPLHv9XNl3uMUWE0yQm4EfkCaKMUVZq2cUd\nzn8XYHHD7+4IXE9e2CwGLgC26CvfrEXsF5M3wL8H7quvfwucR164PkgeDE1iLyEvwv5nYFoCXFr/\n/m6LdXol8Oj699bAdeSF8aXkyfQ2YNuGsa8gL9IB3kFehH2aTLR9CrgbOKDFelkM/BI4BNh8iPvZ\nYcAfgFPJROMhwCjwATLR9lsy2ds0/vbAr+u+8j3yBvs/6t/3Ab8Ctm8Ye4u67RbX/e4E4OHD2M/r\n998zzvQg8PHe6xbb9MF63LweWGeI2/Q84JvARmOUbVTLzmmx3L+px07/tIRMJl0HXNsw9lnAMSx9\nGPY/JEwAACAASURBVHQIebMEmaS5Djh8Gm7LztZ3jfEL4B317z+tx81f9pW/jbxQmG77YWfLXb//\n5BVMr2t6/AP3UH8LyGuBQwbKDyQTeE2Xfe9xpgfJBMfewN4NY3+Beh4kE9ankufI3u/Id+k7T04x\n9qbAhTXW9WRS+RKWXm9cCGw6DWPfBexQ//4emeDpL/8ocFHD2JcBH6h/7wfcAXywr/y9wE9b7Cv7\n1v1ilLyO2KvO4zzgv2vZG6Zh7K8Dp5PnwHWBzwLH17I96jwPahh7bt0vFpPXENuSSc17gHuB3/W2\nd8P43wTOAJ5EXr9dWd9bu/5fvgV8tWHsjcik7H3kNedHgIf1lbe5Pu8d40smmNpcE61NPky7hrz2\nOmCgvM2ydxl7Q/JB/YPAA3W931b/fhD4KrDBdFsvdXv9a9+ynkk+sH5Y02UdiH87sFP9+yzgJOo1\nQP0/fZnm14pfJu9ZXjSwfz+MTFpfDRzbMPYWwEV1210AbFzXTW8fv5q+e94pxj6cTL78LfCP5D3i\nlwa25ZIW63xJ3U/+HthyGNtxCvNuk1focp13th/WGLPqtrxjjPPhHeTv/6yhrOOVuUFn8kT+AO40\nQflOwH0t4o93kd2b/l+Lg+H0uvPPJpMcZ5IZ4Tm1vM2P1Q+Bf6x/71cPjo/1lR8BnNsw9vvrcu4x\n8P4DwBOGsE2XUC/SyR/cHwCPrK8fTl5Yfq1h7IXANvXvy4F3DpS/AfhFi+XeEziavHi8n7wAfHnb\nE0M92e9b/96lnkDf2Fe+D/CrFvG7TPgcT94AP428OfgJ8GNg41o+jB/DrhJKS8gkwzfr9hyt2/eJ\nQ9jPF04Uh7xxWNgw9jFkInzewPutj1Hy5miHvtfr1HXzmPr6lcB103Bbdra+++LP6Xt9f//8yCd9\n907T/bCT5e5b9vFuKB9KKjWMfSfw5Pr3bb2/+8of3+Gyt7oZrnF7v3Mfr/v988na988mz/lHNIx9\nKnkNMHeMsrnkb+o3pmHse1h6EX8rsMsY2/PuFrEfV/8udT9/Ul/5dk1j1+93lsTrOPZdwM59rzes\n62aj+vpNwFUNY3eWBKvxfws8pW+5lwDP6St/FnBDw9ifJm9IX0M+kL2evEbv3VQ2vm4Bzq6xNh14\nf1jX0IfTUQKi49idJXy6XPa6321KdvX0auDb5DX6rcA/Azu23J4LgcfXv28GnjpQviPZSqRJ7DvI\n2p7jlT+bbP3TJPYJ5O/BK8hk/g+A75O1ZOeQyaDPNYz9K+Dlfa+3r+/9O3l+b/vwvtPE6Qrm3SbJ\n1uU672w/rN8/kjyn/wV53bl+nR4H/HndFv88lHXc9UZcXSbypustE5S/Bbi+RfwuL7JvY9mLvAJ8\nkWy2uF2bkwR54bR9/XtWPUk8ta/8icCtLdbL08kfvE8Ca9f3ukiy/Rp4wUD5s4CRhrFHyWq4vfU/\n1kV808RG/3KvTdbQ6D1lvoms2ty0NthYN8L9F8bb0O5mssuEz03Abn2vexfXPyWrpLf9MewyodS/\nTTcF3kc2XVpMPgl9J/CIhrFvpu8iYYzyV5DV3Zsu+z7ACHDgkNfJTcCufa8fVdfTI+rrbcl+sqbb\ntux6ff8GeG79e8u6Tl7aV/48ssnUdNsPO1vu+v1Rsl+tbcaZXtr0+Cdv4I+of/83A7UcyZvjX7ZY\n9s5uhge26c+B/QfK9waubhj7bgYugAfK59M8WdVl7POBv6t//4CBazvyBrZp0uQWlv72b1zX//P7\nyp9ONtNtuj07S+J1HPu3/fsyeVOzmKWtCbaj+fm8syRY/f7gddHd1JvA+nrrFst+w8D+MZussXkO\neQ3T9rrlYPL3uT9RMKxr6M4SEB3H7izh0+Wy03cu73vvscAHyXuYxcD3Wyz3j6gVAsjKAa8aKH9B\n03MXea/4tAnKnw7c1TD2zcAz69+Prutpz77yPYBfN4y9sHdOHFjnV5OVM7Zse
```

Comment:
search is kind of a sensitive topic here at mozilla due to partner agreements, probably not the sort of thing we want to encourage people to just play around with randomly. could you replace this with some other quantity? it doesn't matter much which.